sites/public/pages/listings.tsx

```diff
@@ -52,6 +74,106 @@ export default function ListingsPage(props: ListingsProps) {
       </Head>
       <MetaTags title={t("nav.siteTitle")} image={metaImage} description={metaDescription} />
       <PageHeader title={t("pageTitle.rent")} />
+      <Modal
+        open={filterModalVisible}
+        title={"Filter Results"}
+        ariaDescription={"testing aria"}
+        actions={[
+          <Button
+            onClick={() => setFilterModalVisible(false)}
+            styleType={AppearanceStyleType.primary}
+          >
+            Apply
```

Comment:
Along the same line as Elen's comments, "Apply" and "Close" below should probably be added to general.json and other languages so they  can be translated. Fine with me if you want to add a TODO for that. We should probably have a new "filter" dict with all these filter related things.

Reply:
Agree but we can make these later.  Let's leave them for now.